### PR TITLE
NEPT-193: Improve the suffix behat test

### PR DESCRIPTION
**Pull request by ademarco**
This PR is a follow up of https://github.com/ec-europa/platform-dev/pull/1144 which is now closed.

### CODE REVIEW

File: tests/features/multilingual.feature [hunk 1]
```diff
@@ -56,20 +56,14 @@ Feature: Multilingual features
...
-    And I fill in "Default front page" with "admin/fake-url"
-    And I select "01000" from "edit-classification"
-    And I press the "Save configuration" button
-    Then I should see the success message "The configuration options have been saved."
+    And the site front page is set to "admin/fake-url"
+    And the "en" language "prefix" is set to "en-prefix"
     And the cache has been cleared
-    When I reload the page
-    Then I should be on "admin/config/system/site-information_en-prefix"
+    And I go to "admin/config/system/site-information"
```
Comment (voidtek) [inline tests/features/multilingual.feature]: Should be "When"

**Comment by voidtek** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Comment by drupol** [comment]
:1st_place_medal:

**Pull request merged**